### ISSUE TITLE
Trainning with Ljspeech dataset 12 epoch

### ISSUE THREAD

**Issue by jiqizaisikao** [KinglittleQ/GST-Tacotron#2]
When synthesis eval wav,the result wav is silent and  the attention is meaningless and completely misaligned

**Comment by jiqizaisikao** [comment]
mag loss is about 0.4 and mel loss is about 0.4

**Comment by KinglittleQ** [comment]
@jiqizaisikao Did you modify any code or hyperparameters? If you did, please show me the changes.

**Comment by KinglittleQ** [comment]
@jiqizaisikao In fact, GST-Tacotron is a model trained on a multi-speaker dataset. It would be better to use Tacotron rather than GST-Taocotron if you want to train on LJSpeech dataset. [Here](https://github.com/KinglittleQ/Tacotron) is my Tacotron repository.

**Comment by jiqizaisikao** [comment]
I just modify your temp code to train with Ljspeech date set, set sr=22050 n_fft=768 hop_length=192 batch_size=8 lr=0.0001
